.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -669,6 +669,32 @@ popd > /dev/null
 chown -R ${DAEMON_USER}:${COURSE_BUILDERS_GROUP} ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 chmod -R 555 ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 
+################################################################################################################
+################################################################################################################
+# BUILD AND INSTALL ANALYSIS TOOLS TS
+
+echo -e "Build and install analysis tools ts"
+
+mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS
```

Comment:
```suggestion
mkdir -p "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
```

Reply:
Is the quotations needed. I copied the AnalysisTools build procedure. https://github.com/Submitty/Submitty/blob/main/.setup/INSTALL_SUBMITTY_HELPER.sh#L587-L627

Reply:
Yes, it is good practice to quote variables in shell scripts to prevent command injection.  I actually have [another PR ](https://github.com/Submitty/Submitty/pull/7880) open right now to quote a bunch of variables in this file.